src/GraphQL/Language/CoreToVanillaConverter.cs

```diff
@@ -15,425 +13,21 @@ namespace GraphQL.Language
     /// <br/>
     /// For more information see https://spec.graphql.org/June2018/#sec-Language.Document.
     /// </summary>
-    public static class CoreToVanillaConverter
+    public class CoreToVanillaConverter : DefaultNodeVisitor<CoreToVanillaConverterContext>
```

Comment:
Now this class holds only 2-3 lines of code and may be completely removed. But in general we should inspect parsed document for definitions other that executable definitions and throw error (to be spec complient).